src/Invictus.Testing/Converter.cs

```diff
@@ -1,86 +1,74 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.Linq;
-using System.Net.Http;
-using System.Text;
-using System.Threading.Tasks;
 using Invictus.Testing.Model;
 using Microsoft.Azure.Management.Logic.Models;
-using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
 
 namespace Invictus.Testing
 {
-     public class Converter
+    internal class Converter
```

Comment:
Maybe emphasize what we are converting?
```suggestion
    internal class LogicAppConverter
```

Reply:
Could be, but an prev conversation talks about maybe we change the result types and use the SDK models direclty so we don't need this converter anymore.

Reply:
Done.